### ISSUE TITLE
Displaying 404 error pages using TwigBridge

### ISSUE THREAD

**Issue by iamkirkbater** [rcrowe/TwigBridge#216]
I had a little problem trying to figure out how to override the default `[errorCode].blade.php` files to use a Twig template.  I have wrote a little forum post on Laravel.io [here](http://laravel.io/forum/08-11-2015-using-twigbridge-to-display-server-error-pages), but I was wondering if you wanted to add this to the Readme.md file?  I already have it written up, but I don't want to make a PR unless you think it's something that would be of value.  

Basically I'm just not sure if you want to put it in the main Readme or if you just want to leave it as a blog post type thing on how to do it.

Thanks,
Kirk

**Comment by barryvdh** [comment]
`errors/404.twig` works fine if `errors/404.blade.php` doesn't exist, if you have registered the ServiceProvider correctly.
